MediumDifficultyProgram6/RevString/RevString.java

```diff
@@ -0,0 +1,27 @@
+package RevString;
+
+public class RevString {
+    /*
+    Q6. Write a program to reverse the string without changing the position of special character.
+ */
+    public static void main(String[] args) {
+        String result = Reverse("Ro@hitshetty@");
+        System.out.println(result);
+    }
+
+    static String Reverse(String inputString) {
+        String result = "";
+        String specialCharatcterList = "@,#,$,<,>,?,/,^,&,*,(,),{,},[,],:,.";
```

Comment:
Do not use this as static. write the code for this.

Reply:
replaced static values for special characters and used code